scripts/lib/mkcloud-libvirt.sh

```diff
@@ -57,36 +57,85 @@ function libvirt_net_start()
         ip link set mtu 9000 dev $dev
     done
 
+    boot_mkcloud=/etc/init.d/boot.mkcloud
+    boot_mkcloud_d="$boot_mkcloud.d"
+    boot_mkcloud_d_cloud="$boot_mkcloud_d/$cloud"
+
     if [ -z "$NOSETUPPORTFORWARDING" ] ; then
+        # FIXME: hardcoded assumptions about admin net host range
```

Comment:
While trying to update the same, I have realized that the ip addresses are not exactly hard-coded but are coming from ``network.json`` most probably due to this: 
` "dhcp": {
                            "end": "192.168.54.80",
                            "start": "192.168.54.21"
                        },
`
So for changing these addresses, we would have to talk to Crowbar. There could be a way to do so already from mkcloud or qa_crowbarsetup.sh but looking a these scripts closely did not leave any clues from my side.